app/models/group.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+#
+# Model for Housing Groups
+#
+# @attr [Integer] size The room size that the Group wants.
```

Comment:
So I think you've been doing YARD wrong 😕 - I'm pretty sure the format is `@attr attr_name [Type] description`, and until now the attribute name has been going _after_ the type. Let's fix all the new comments in this branch and I'll open an issue to update all our earlier documentation at some point.